netbout-web/src/test/java/com/netbout/email/EmAliasTest.java

```diff
@@ -103,4 +103,41 @@ public void doesNotSendMailToAuthor() throws Exception {
             )
         );
     }
+
+    /**
+     * EmAlias can send an email with address confirmation link.
+     * @throws Exception If there is some problem inside
+     */
+    @Test
+    public void sendsConfirmationEmail() throws Exception {
+        final Postman postman = Mockito.mock(Postman.class);
+        final Alias alias = new EmAlias(new MkBase().randomAlias(), postman);
+        alias.email("mihai@test.com", "netbout.com/test/verification/link");
+        final ArgumentCaptor<Envelope> captor =
+            ArgumentCaptor.forClass(Envelope.class);
+        Mockito.verify(postman).send(captor.capture());
+        final Message msg = captor.getValue().unwrap();
+        MatcherAssert.assertThat(msg.getFrom().length, Matchers.is(1));
+        MatcherAssert.assertThat(
+            msg.getSubject(),
+            Matchers.equalTo("Netbout email verification")
+        );
+        MatcherAssert.assertThat(
+            msg.getAllRecipients()[0].toString(),
+            Matchers.containsString("mihai@test")
+        );
+        final ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        MimeMultipart.class.cast(msg.getContent()).writeTo(baos);
+        final String content = new String(baos.toByteArray());
```

Comment:
@amihaiemil this will use the default encoding which is prone to build failure, please provide explicitly the encoding